### PR TITLE
4293 11 table individual inbox

### PR DESCRIPTION
**Pull request by matthopson**
Screen Shot 2020-06-04 at 3 11 03 PM

### CODE REVIEW

File: shared/src/business/entities/CaseMessage.js [hunk 1]
```diff
@@ -49,9 +56,24 @@ CaseMessage.validationRules = {
...
+    .description('The status of the associated case.'),
   createdAt: joiStrictTimestamp
     .required()
     .description('When the message was created.'),
+  docketNumber: joi
+    .string()
+    .regex(DOCKET_NUMBER_MATCHER)
+    .required()
+    .description('The docket number for the associated case.'),
+  docketNumberSuffix: joi
```
Comment (rachaelparris) [inline shared/src/business/entities/CaseMessage.js]: This should probably be `docketNumberWithSuffix`, same as we are doing on the work items - is there any reason we need the suffix by itself?

File: shared/src/business/useCases/messages/createCaseMessageInteractor.js [hunk 2]
```diff
@@ -31,6 +31,15 @@ exports.createCaseMessageInteractor = async ({
     throw new UnauthorizedError('Unauthorized');
   }
 
+  // TODO: Would it be better to just pass this in since case detail is already loaded in the action?
```
Comment (rachaelparris) [inline shared/src/business/useCases/messages/createCaseMessageInteractor.js:34]: The user data is already loaded in too, but we are also retrieving it in this interactor. I'm not sure what the best approach is honestly, and would like more input!

File: web-client/src/views/Messages/CaseMessagesIndividualInbox.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+              <th>From</th>
+              <th>Section</th>
+              <th>Attachments</th>
+            </tr>
+          </thead>
+          <tbody>
+            {formattedMessages.map((message, idx) => {
+              return (
+                <tr key={idx}>
+                  <td>Docket</td>
```
Comment (rachaelparris) [inline web-client/src/views/Messages/CaseMessagesIndividualInbox.jsx]: This should probably be `{message.docketNumberWithSuffix}`

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/entities/CaseMessage.js (hunk 1 above)
Comment (matthopson) [reply shared/src/business/entities/CaseMessage.js]: Ah, I saw docket number and docketNumberSuffix on WorkItem and went with that.

File: shared/src/business/useCases/messages/createCaseMessageInteractor.js (hunk 2 above)
Comment (matthopson) [reply shared/src/business/useCases/messages/createCaseMessageInteractor.js:34]: I generally think it's better to do queries in the interactor. It adds an additional layer of security, imo.

**Review by kkoskelin — APPROVED** [review]
Nice.  Thanks for the additional test coverage on the newly-exported functions, too.

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**